R/target_market_share.R

```diff
@@ -235,8 +235,11 @@ target_market_share <- function(data,
 
   data %>%
     pivot_longer(cols = starts_with("weighted_")) %>%
+    filter(!is.na(.data$value)) %>%
```

Comment:
@jdhoffa, can you confirm this is reasonable? It seems to me but I'd like to double check as it may result in the loss of data.